src/prg_system_mod.F90

```diff
@@ -2658,26 +2658,30 @@ subroutine prg_graph2bml(graph,bml_type,g_bml)
     allocate(row(nats))
 
     mdim = bml_get_m(g_bml)
-
+    write(*,*)"mdim",mdim
     !$omp parallel do default(none) private(i) &
     !$omp private(ii,row) &
     !$omp shared(nats,g_bml,graph,mdim)
     do i = 1, nats
       ii = 1
       row = 0.0
       do while (graph(ii,i).gt.0)
-        row(graph(ii,i)) = 1.0
+        row(graph(ii,i)) = 1.0_dp
         !       call bml_set_element_new(g_bml,i,graph(ii,i),1.0)
         !       call bml_set_element_new(g_bml,graph(ii,i),i,1.0)
         ii = ii+1
       enddo
       call bml_set_row(g_bml,i,row,0.5_dp)
     enddo
     !$omp end parallel do
-
+!    stop
```

Comment:
Why the commented out stop here? Is that a leftover debug statement?

Reply:
Should be fixed now